api/graphql/schema.graphql

```diff
@@ -1439,6 +1439,7 @@ type Query {
   ): [CommunityInterest]!
     @guard
     @paginate(defaultCount: 10, maxCount: 500, scopes: ["authorizedToView"])
+  developmentPrograms: [DevelopmentProgram]! @all
```

Comment:
Is it necessary to add this?  Could you pull it down from the Community model instead?

Reply:
Oops, you're right.  It should be the set of development programs associated directly with the talent nomination event. :+1:

Reply:
Okay this should be fixed by [1dbefdf](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13111/commits/1dbefdf47158728cdeec25b784fef1d5bd8ea689)